region/brinstar/pink/Mission Impossible Room.json

```diff
@@ -929,6 +945,53 @@
       "clearsObstacles": ["B"],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 4],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) (Unbroken Super Block)",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"obstaclesCleared": ["A"]},
+        {"or": [
+          "h_CrystalFlash",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"resourceMissingAtMost": [{"type": "Super", "count": 0}]},
+            {"or":[
+              {"and": [
+                "canBePatient",
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 5}}
+              ]},
+              {"and": [
+                "canBeExtremelyPatient",
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+              ]}
+            ]}
+          ]}
+        ]},
+        "canPauseAbuse",
+        {"or":[
+          {"canShineCharge": {"usedTiles": 16, "openEnd": 0}},
+          {"and":[
+            {"spikeHits": 1},
+            "canUseIFrames",
+            {"canShineCharge": {"usedTiles": 23, "openEnd": 0}}
+          ]}
+        ]},
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt"
+      ],
+      "resetsObstacles": ["R-Mode"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Kill both Sidehoppers for energy, or else Crystal Flash (kill both Sidehoppers anyway so you have the runway free).",
+        "To R-Mode Spark Interrupt from spikes, damage down to within 1 more spike hit of health and set reserves to Manual.",
+        "Gain shinecharge, press Start and immediately step on spikes during the fadeout. At the last moment before the game",
+        "pauses, begin spark windup. In the pause menu, set reserves back to Auto but do not refill any energy (unpause with",
+        "00 energy). As soon as you hear the reserve fill complete, pause and unpause again to restore visibility."
```

Comment:
I'd like you to write this description into the `canRModeSparkInterrupt` tech description instead.  I don't think it is different enough to be an extension tech and we can attach movement type tech where needed. (`canRModeSparkInterrupt` is easy enough to do at lower difficulties, but there are so many additional requirements, like `canCarryBlueSuit`, that it may need to be placed high)

This probably works with any kind of pause abuse too, enemy hit/heat/spike.

Reply:
This can be a follow-up PR, I believe